DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
@makortel, thanks for pointing this out. Indeed, it might be better to use `std::string` as the key type but then all of the downstream code needs to be appropriately updated. Also note that we would not be able to use `operator[]` to access the map elements, since the map needs to be `const`. Instead, map elements would have to be accessed using `at()`.

@mverzett, not being a map expert, I'm reading online that `map` vs `unordered_map` performance can depend on the map size and the exact implementation of the `unordered_map` so to me it is not clear what would work best in this case. However, I do agree that not having to worry about the variable order would make the code less error prone so using maps should generally be better than what is currently being used. Hence, if you feel like implementing maps, go for it :)